base/frontend/sourcegraph-frontend.Ingress.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
     alb.ingress.kubernetes.io/actions.ssl-redirect: '{"Type": "redirect", "RedirectConfig": { "Protocol": "HTTPS", "Port": "443", "StatusCode": "HTTP_301"}}'
     alb.ingress.kubernetes.io/scheme: internet-facing
     alb.ingress.kubernetes.io/tags: Environment=production,Team=tooling
+    alb.ingress.kubernetes.io/certificate-arn: arn:aws:acm:us-west-2:752180062774:certificate/a57e1dd4-bffd-4c78-ad34-db480408eec0,arn:aws:acm:us-west-2:752180062774:certificate/b7a45f0c-3009-42c2-97b9-ab81a61d1b25
```

Comment:
this is annoying to have to do on every ing (and I forgot initially). Maybe there's a flag for this